sqrl-planner/src/main/java/com/datasqrl/engine/server/VertxEngineFactory.java

```diff
@@ -40,9 +40,13 @@ public Class<? extends IExecutionEngine> getFactoryClass() {
   public static class VertxEngine extends GenericJavaServerEngine {
 
     @Inject
-    public VertxEngine(PackageJson packageJson, ObjectMapper objectMapper) {
+    public VertxEngine(PackageJson packageJson) {
       super(
-          ENGINE_NAME, packageJson.getEngines().getEngineConfigOrEmpty(ENGINE_NAME), objectMapper);
+          ENGINE_NAME,
+          packageJson
+              .getEngines()
+              .getEngineConfig(ENGINE_NAME)
+              .orElseGet(() -> new EmptyEngineConfig(ENGINE_NAME)));
```

Comment:
I added `.getEngineConfigOrEmpty(ENGINE_NAME)` exactly to eliminate this `orElseGet(...)` boilerplate, why revert?